index.rst

```diff
@@ -9,9 +9,6 @@ XCORE |reg| -VOICE Solutions
 
   doc/quick_start_guide/index
   doc/programming_guide/index
-  modules/voice/doc/user_guide/audio_processing/index
```

Comment:
Please reverse this change and the two immediately below.

Reply:
Done in https://github.com/xmos/sln_voice/pull/367/commits/c64b2edb12515936abc8ebe0196fd3db07a2d64a.